app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] Kindly remove this line of code from this file `belongs_to :author, class_name: 'User'`. The two classes are connected through the _**foreign_key**_ `user_id` in the `Comments` table. 😃 🖥️

Reply:
Fixed.